lib/name_drop/configuration.rb

```diff
@@ -15,18 +15,4 @@ def initialize
       @access_token = 'NotARealAccessToken'
     end
   end
-
-  class << self
```

Comment:
I know that these methods belong to the `NameDrop` module, and not the `NameDrop::Configuration` class.  However, I liked logically grouping them into the `lib/name_drop/configuration.rb` file that contains all the other `configuration`, as I thought it was less confusing than adding them into the `lib/name_drop.rb` file.